packages/atomic/cypress/utils/setupComponent.ts

```diff
@@ -29,20 +36,18 @@ export function setupIntercept() {
 
 export function setUpPage(htmlCode: string) {
   setupIntercept();
-  // Setup page with new component
   cy.visit('http://localhost:3333/pages/test.html');
   cy.injectAxe();
   injectComponent(htmlCode);
-  cy.wait(1000);
+  cy.wait(500); // TODO: waiting for @coveoSearch is less flaky
```

Comment:
Doing this requires modifying a bunch of tests, it would be too big a refactor for this PR.
If you want an idea https://github.com/coveo/ui-kit/pull/667/files